pex/variables.py

```diff
@@ -33,11 +33,40 @@ def iter_help(cls):
       variable_type, variable_text = cls.process_pydoc(getattr(value, '__doc__'))
       yield variable_name, variable_type, variable_text
 
-  def __init__(self, environ=None, rc='~/.pexrc', use_defaults=True):
+  @classmethod
+  def from_rc(cls, rc=None):
+    """Read pex runtime configuration variables from a pexrc file.
+
+    :param rc: an absolute path to a pexrc file.
+    :return ret_vars: a dict object containing key values pairs found in processed pexrc files
+    """
+    ret_vars = {}
+    rc_locations = ['/etc/pexrc',
+                    '~/.pexrc',
+                    os.path.join(os.path.dirname(sys.argv[0]), '.pexrc'),
+                    os.path.join(os.getcwd(), '.pexrc')]
```

Comment:
did this actually end up being needed given the new `--rcfile` flag? if so, why?

seems to me like tests were passing without it before?